### PR TITLE
test: add tests using mock app

### PR DESCRIPTION
**Pull request by TheUnderScorer**
Example run: https://github.com/fingerprintjs/fingerprintjs-pro-cloudflare-worker/actions/runs/8108181033/job/22160930702

### CODE REVIEW

File: .github/workflows/teste2e.yml [hunk 1]
```diff
@@ -19,6 +19,73 @@ on:
           - patch
 
 jobs:
+  build-and-deploy-and-test-e2e-mock:
+    runs-on: ubuntu-20.04
+    name: Build & Deploy & Test e2e using mock app
+    env:
+      FPCDN: mock-warden.fpjs.sh
```
Comment (necipallef) [inline .github/workflows/teste2e.yml]: Let's use GitHub secrets here and the line below

Comment (TheUnderScorer) [reply .github/workflows/teste2e.yml]: Good point, I'll ask Ilya

Comment (TheUnderScorer) [reply .github/workflows/teste2e.yml]: Done!

**Review by necipallef — APPROVED** [review]

**Comment by necipallef** [comment]
Approved. One tiny suggestion: it would be more clear to use `FPCDN_URL` instead of `FPCDN` (same for Ingress).

**Review by sshelomentsev — APPROVED** [review]

**Comment by TheUnderScorer** [comment]
> Approved. One tiny suggestion: it would be more clear to use FPCDN_URL instead of FPCDN (same for Ingress).

Good idea! But I think that we should implement this in a separate PR

**Pull request merged**